types/base/api.d.ts

```diff
@@ -0,0 +1,42 @@
+type Part = {
+    name: string;
+    material: string;
+    color: string;
+}
+
+type Spec = {
+    brand: string;
+    model: string;
+    parts: Array<Part>;
+    initials: string;
+    engraving: string;
+    initialsExtra: object;
+    variant?: string;
+    version?: string;
+    description?: string;
+};
+
+type ProductionOrderOptions = {
+    brand?: string;
+    model?: string;
+    variant?: string;
+    parts?: Array<Part>;
+    initials?: string;
+    engraving?: string;
+    initialsExtra?: object;
+    gender?: string;
+    size?: number;
+    pending?: boolean;
+    notify?: boolean;
+    productId?: string;
+    currency?: string;
+    country?: string;
+    meta?: Array<string>;
+}
+
+export declare class RipeAPI {
+    constructor(options?: any);
+    authKeyP(key: string, options?: object): Promise<void>;
+    importOrderP(ffOrderId: string, options?: ProductionOrderOptions, callback?: Function): Promise<void>;
+    _queryToSpec(query: string): Spec;
+}
```

Comment:
Fix this newline issue with `cleanup .`

Reply:
Weird, cleanup is not analyzing the `types` folder.

Reply:
Neither `ts` files. I think the extensions are missing in the `admin-scripts` package.

Reply:
I guess you can consider a PR for admin-scripts then.

In any case, you can obviously just add the newlines manually to move on